pages/security/output_encoding.md

```diff
@@ -0,0 +1,43 @@
++---
++title: Output Enco
++permalink: /security/output-enciding
++layout: docs
++sidenav: security
++---
+
+# Output encoding
+
+## What is cross site scripting (XSS)?
+
+Cross site scripting or XSS is a form of attack on a web application which involves executing code on a user's browser. Output encoding is a defense against XSS attacks.
+
+To get a more extensive understanding of XSS, see [excess xss](https://excess-xss.com/).
+
+## How to correctly encode output
+
+Protecting from XSS attacks requires developers to consider how data is being displayed on a page. If the data could come from a user's input in any way (including through the site's URL), then correct encoding of the output has to be considered.
+
+Since most web applications at 18F are built through JavaScript or backend frameworks, this guide will go over output encoding issues by those frameworks in addition to plain JavaScript.
+
+### Plain JavaScript
```

Comment:
I feel like I usually see JS sans frameworks referred to as "vanilla JavaScript," but I don't know if that's necessarily clearer.

Reply:
Yea I usually see that too, will change.